File: C/odds.c
#include <stdio.h>

int main() {
    int num;
    printf("How many odd numbers you wanna print? ");
    scanf("%d", &num);
    int i = 0;
    int acc = 1;

    while (i < num){
        if (acc % 2 != 0) {
            printf("%d ", acc);
            acc++;
            i++;
        }
        else {
            acc++;
        }
    }

    printf("\n");
    return(0);
}

File: Haskell/odds.hs
odds :: Int -> [Int]
odds n = take n [x | x <- [1..], odd x]
  
main :: IO ()
main = do
  putStrLn "How many odds numbers do you wanna print? "
  input <- getLine
  let num = read input :: Int
  putStrLn $ "The first " ++ show num ++ " odd numbers are: " ++ show (odds num)